combat-trainer.lic

```diff
@@ -1177,6 +1177,9 @@ class TrainerProcess
 
     @forage_item = settings.forage_item
     echo("  @forage_item: #{@forage_item}") if $debug_mode_ct
+
+    @analyze_retry_count = settings.analyze_retry_count
```

Comment:
There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.  I'm open to suggestions